### ISSUE TITLE
Standardize Alg2 coefficients

### ISSUE THREAD

**Issue by schanzer** [bootstrapworld/curriculum#2119]
**Linear Models: f(x)=m(x-h) + k**
- [x] Pyret (we don't refer to these)
- [x] Desmos (there isn't one) - add one for horizontal shift
- [x] Lesson plan (bridge between b and k)
- [x] workbook pages (all but the scaffolded page never use b or k)

**Quadratic Models: f(x)=a(x-h)^2 + k**
- [x] Pyret (we don't refer to these)
- [x] Desmos (already this way) -  but we should switch `f(x)` and `g(x)`
- [x] add horizontal shift slide
- [x] Lesson plan 

workbook pages
- [x] Refocus Vertex Form box on `Modeling Fuel Efficiency v. Speed` 
- [x] On Graphing Quadratic Models, switch `f(x)` and `g(x)` for greater consistency.
- [x] While the `Exploring Fuel Efficiency` workbook page uses `f(x)` `lr-plot` actually returns `y=` and we don't have students save this in pyret. Should we change the workbook page to use `y` and `y(x)`?
- [x] None of the definitions on `What kind of model` use this form. Should we change that?

_NOTE: I had actually already avoided using b or k in the workbook for this lr-plot. Maybe we could use this strategy consistently throughout our materials!_ <img width="929" alt="Screenshot 2024-07-16 at 12 59 51 PM" src="https://github.com/user-attachments/assets/5f1657d2-9eee-4bba-95c6-1ef7d238e992">

- [x] add  /vertical shift for y-intercept

**Exponential Models: f(x) =a*b^(x-h) + k**
- [x] Pyret (we don't refer to these)
- [x] Add a slide with just an h slider to Desmos.
- [x] Address why we are assuming the horizontal shift in the model for this data will be zero in the lesson plan. Or introduce h in the lesson plan. 
- [ ] Noting that the quadratic model, exponential model for MA and exponential model for VT all are defined as f(x) in Desmos. Currently leaning toward thinking this is ok since they get renamed when used outside of this file?
- [x] update slide numbers in lesson plan Desmos directions
- [x] update slide numbers in workbook pages
Workbook pages:
- [ ] Fix exponent in lesson plan
<img width="567" alt="Screenshot 2024-07-19 at 4 48 56 PM" src="https://github.com/user-attachments/assets/666a5b5f-7e61-43ad-90df-1692e73d73a9">

**Logarithmic Models: f(x) = a*log_b(x-h) + k**
- [x] update slide numbers in lesson plan Desmos directions
- [x] Address why we are assuming the horizontal shift in the model for this data will be zero.
- [x] Pyret - decided that adding - 0 here would add unnecessary complexity to the code and be counterproductive.

> # exponential : Number -> Number
> # change the function below to be your best exponential model
> fun exponential(x):  (1 * num-expt(2, (~1 * (x - 0)))) + 3 end

Desmos 
- [x] the activity description and slides 4 and 5 still used c instead of k.
- [x] we had a and b reversed on one slide
- [x]  For consistency, added a slide with an h slider
- [x] This was the only file that included a linear model slider activity (forcing the question of whether we use b or k) Use of slide 1 is optional, so we cut it.
- [x] slide 6 uses `f(x)=mx + b` for the transformed line... Decided we have to leave it because there's already enough cognitive load
- [x] change c to k on *transforming data*

**Periodic Models: f(x) = asin(b(x-h)) + k**
- [x] Pyret - add standard periodic function scaffolding to these definitions.
-- fun periodic(x): x end
-- fun hybrid(x): x end
- [x] Desmos (already was)
- [x] fix coefficients in modeling the ferris wheel data - still refers to c and d 
- [x] find and replace h-shift to horizontal shift and v-shift with vertical shift
- [x] fix slide numbers in lesson plan and on pages

**Comment by flannery-denny** [comment]
@schanzer I didn't get as far with this as I'd hoped. Given that our slider explorations are already pretty laborious and long, I'm not sure that adding `h` in a meaningful way is useful.

I'm also struggling with how to write (x-h) as an exponent in asciidoc. I've pushed [this commit](https://github.com/bootstrapworld/curriculum/commit/139bd701b3dff97ef2042e4540275925739a9b72) which I mistagged to 2092.

Perhaps you can help with the exponent.
Also curious what you think about the beginnings of wording I pushed declaring that h is in the standard form, but is commonly zero so we won't be investigating it.

**Comment by flannery-denny** [comment]
@schanzer Here's a draft of horizontal and vertical shift sliders for you to react to the utility of. 

- I imagine the linear stuff would get added to the linear lesson. 
- If you think there's a point to the exponential stuff, we could add similar things to each of the other lessons. The purpose of these could either be to:
-- establish that vertical shift does not correspond to horizontal shift
-- identify the values for which horizontal and vertical shift can be left out of the equation

**Comment by schanzer** [comment]
@flannery-denny thanks for drafting this!

- Seeing the tables is very distracting. Can you stick those in a folder and make it invisible?
- Having both lines be movable makes this much more complex than it needs to be. For each of these slides, there should be a single "reference line" and a second "movable line". In the movable case, students should be able to change both the horizontal and vertical shift, and challenged to match the reference line using _only_ one slider, then _only_ the other slider, then _both_ sliders. In the linear case, we can ask them to write down the values and see if they can find the relationship between _h_ and _b_. 
- I don't think it's necessary to ask them which values allow these coefficients to be left out: it's going to be "when they are zero" every time for the non-linear cases.

**Comment by flannery-denny** [comment]
@schanzer [Here's a new draft](https://teacher.desmos.com/activitybuilder/custom/6697de266df88d2584d1f41c)
I've incorporated your feedback and made 5 slides focused on linear functions. If you like these, I'll add them to the linear modeling lesson.  

Your suggestion for curves didn't make sense, because there's now way to hide the function definition for the reference line. I've proposed an alternative. If you think these are useful, I'll make the comparable slides for the other kinds of models.

Interesting that you found the tables distracting. I intentionally sought out a way to reveal the tables, because I wanted to be able to look at the patterns through that lens. And, if you don't show the tables, the lines don't have any points marked, which sometimes makes it hard to tell whether functions actually are lining up. I've added a note to make them optional - does scratch the itch?

**Comment by schanzer** [comment]
@flannery-denny you can hide any folder you want. I would make a folder for each slide, and then put the reference curve and the associated tables inside of it. Then hide that folder from kids.

I still don't like the fact that you have two different movable lines. Let's just have ONE line with both sliders active, and challenge kids to match the reference line using each slider individually. "Can you match the reference line using only _h_? Only _k_? Do you need to use _both_?

**Comment by schanzer** [comment]
@flannery-denny just noticed that you wanted me to look at the DRAFT, not the published exercise. Cool! Just one movable function.

I went ahead and copied slide 7, so that slide 8 could be an example of what I mean.

**Comment by flannery-denny** [comment]
@schanzer I've updated the Desmos deck to include slides like the one you made for each of the other function types and moved my two exponential function explorations to the end of the deck for reference since they're attempting to do something different.

The challenge and directions on the slide you made don't really capture my interest and I'm not clear why all three of those questions are really worth students time. Look forward to chatting about them or seeing what you do with them. I've added incorporating discussion of horizontal shift into lesson plans to #2092 and think any further conversation about this deck belongs over there as it doesn't really fit under the scope of this issue.
